src/api_routes_http.rs

```diff
@@ -352,6 +354,12 @@ pub fn config(cfg: &mut web::ServiceConfig, rate_limit: &RateLimitCell) {
           .route("/community", web::post().to(route_post::<PurgeCommunity>))
           .route("/post", web::post().to(route_post::<PurgePost>))
           .route("/comment", web::post().to(route_post::<PurgeComment>)),
+      )
+      .service(
+        web::scope("/admin/review_comments")
```

Comment:
This file needs some rearranging. Seems like both this and purge need to move up to the admin scope. 

Also lets future proof this in case we want to add other reviewable items:

- admin
  - review
    - comment
      - list
      - approve